Acknowledge community triagers in area-owners.md (#59748)
dotnet/runtime: docs/area-owners.md

|------------------|---------------|-----------------------------------------------------|--------------|
| arch-wasm        | @lewing       | @lewing @BrzVlad                                    |              |

## Community Triagers

> The repo has a number of community members carrying the triager role. While not necessarily associated with a specific area they might be able to assist with labeling certain issues and pull requests. Currently, the following community members are triagers:

Reviewer: ```suggestion
The repo has a number of community members carrying the triager role. While not necessarily associated with a specific area, they may assist with labeling issues and pull requests. Currently, the following community members are triagers:
```
